graph_algorithms/algorithms/examples/conn_comp2.gsql

```diff
@@ -1,4 +1,4 @@
-CREATE BATCH QUERY conn_comp2 () FOR GRAPH social {
```

Comment:
What is conn_comp2?
It seems it is not new but it is not documented.

Reply:
It is for test and it will be removed.